mflowgen/Tile_MemCore/construct.py

```diff
@@ -78,7 +79,7 @@ def construct():
 
   info           = Step( 'info',                           default=True )
   #constraints    = Step( 'constraints',                    default=True )
-  dc             = Step( 'synopsys-dc-synthesis',          default=True )
+  genus          = Step( 'cadence-genus-synthesis',        default=True )
```

Comment:
Seems like we could do this in a way that makes it easy to switch between the two. Instead of calling the step dc or genus, we could just call it synthesis, and use a parameter to choose which one we want to use.

Reply:
This probably deserves a bigger discussion at the meeting today about whether we should really attempt to keep things tool-agnostic here. Chris was suggesting that it might be difficult. For example, the new custom-genus-scripts I added are Genus-specific (they rely on the particular format of the sdc file names used by Genus). We'll need to add a bunch of switches to make sure we don't use Genus scripts with DC and vice versa ... seems annoying.

Reply:
I've changed the name of this variable from `genus` to `synth`, I think that's a decent start and we can do more later.